turbo/app/snapshots_cmd.go

```diff
@@ -641,3 +691,113 @@ func doUploaderCommand(cliCtx *cli.Context) error {
 	}
 	return err
 }
+
+/*
+func doBodiesDecrement(cliCtx *cli.Context) error {
```

Comment:
do we sometimes comment out old code and leave it in because we think we may use it again in the future or is this something that can be removed in general?

Reply:
usually we removing it. 
but this code is "The Data Fix" (most important part of PR :-) ) which required to fix some broken `bodies.seg` files (only for mainnet) - but because nobody will run this code but me (then I will release updated files and never run it again)... 
maybe i just need move this code into `./cmd/integration` or `./cmd/hack` - this tools are made for "dangerous - don't run it if you don't understand what you are doing" cases - and uncomment it there.